docs/cloud/azureai/sharing-your-flows-in-promt-flow-gallery.md

```diff
@@ -0,0 +1,122 @@
+# Sharing Your Flows in Prompt Flow Gallery
+
+:::{admonition} Experimental feature
+This is an experimental feature, and may change at any time. Learn [more](../../how-to-guides/faq.md#stable-vs-experimental).
+:::
+
+In this document, we will walk you through the steps to share your flows in Prompt Flow Gallery. This needs to register your flow as a model with specific flow metadata in [azureml registry](https://ml.azure.com/registries/azureml/models?tid=72f988bf-86f1-41af-91ab-2d7cd011db47). Once completed, the model will be shown as a flow example in Prompt Flow Gallery on the workspace portal page.
+
+## Passing CELA review of your flow
+
+As your flow needs to be registered in system registry and shared globally, it requires a CELA pass review. Please contact PM: chenjieting@microsoft.com to go through the CELA review process.
+
+## Preparing your flow for sharing
+
+### Creating a `README.md` file for your flow
+
+To make the flow easily understandable, include a `README.md` file in the flow folder explaining how to use it. The README might contain the following sections:
+- The purpose of the flow.
+- The tools utilized in the flow.
+- The prerequisites required to execute the flow.
+- The knowledge that can be acquired from this flow.
+- Any additional information about your flow.
+
+Refer to [this example README](https://github.com/Azure/promptflow/blob/main/examples/ask_wikipedia/README.md) with sections addressing points 1 to 4. A well-written README improves discoverability and encourages collaboration.
```

Comment:
Could remove this sentence for now. Community may not have access to azure/pf repo